content/blog/provisioned-concurrency-insurance-against-cold-starts-in-aws-lambda/index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Provisioned Concurrency: Insurance Against Cold Starts in AWS Lambda"
+date: 2019-12-18
+meta_desc: "Using Pulumi to configure AWS Lambda provisioned concurrency"
+meta_image: meta.png
+authors: ["mikhail-shilkov"]
+tags: ["AWS", "Serverless"]
+---
+
+AWS has recently [announced](https://aws.amazon.com/blogs/aws/new-provisioned-concurrency-for-lambda-functions/) the launch of **Provisioned Concurrency**, a new feature of AWS Lambda that intends to solve the problem of cold starts. In this article, we take another look at the problem of latency-critical serverless applications, and how the new feature impacts the status-quo.
+<!--more-->
+
+## Concurrency Model of AWS Lambda
+
+Despite being serverless, AWS Lambda uses lightweight containers to process incoming requests. Every container, or worker, can process only a single request at any given time.
+
+{{< figure src="./executions.png" caption="Overlapping executions land on separate workers" >}}
+
+Therefore, the number of concurrent requests define the number of required workers that a specific AWS Lambda needs to serve at any given moment.
```

Comment:
```suggestion
Because of this, the number of concurrent requests defines the number of required workers that a specific AWS Lambda function needs to serve a response at any given moment.
```

Reply:
@spara Are resolved suggestions applied or discarded?

Reply:
They should be applied, fixing these now.